Sources/Sentry/Public/SentryMechanism.h

```diff
@@ -58,7 +58,7 @@ SENTRY_NO_INIT
  * Information from the operating system or runtime on the exception
  * mechanism.
  */
-@property (nullable, nonatomic, strong) SentryMechanismMeta *meta;
+@property (nullable, nonatomic, strong) SentryMechanismContext *meta;
```

Comment:
`m`: This gets serialized as `meta` key. I think we should keep meta in the name. What about `SentryMechanismMetaInformation`?